weggli_native-config.cmake.in

```diff
@@ -0,0 +1,19 @@
+@PACKAGE_INIT@
```

Comment:
Mind putting this under a `cmake/` subdirectory?

Reply:
Done|